server/netty-server/src/main/scala/sttp/tapir/server/netty/package.scala

```diff
@@ -25,4 +37,53 @@ package object netty {
       run(routes.toList)
     }
   }
+
+  implicit class RichOptionalNettyResponse(val r: Option[NettyResponse]) {
+    def handle(
+        byteBufHandler: (ByteBuf) => Unit,
+        chunkedInputHandler: (HttpChunkedInput, Long) => Unit,
+        noBodyHandler: () => Unit
+    ): Unit = {
+      r match {
+        case Some(value) =>
```

Comment:
[minor] you can do `case Some(Left(...)) => ...` etc.

Reply:
Good idea, thanks!